appservice/src/FileUtilities.ts

```diff
@@ -12,6 +12,14 @@ export function getFileExtension(fsPath: string): string | undefined {
     return fsPath.split('.').pop();
 }
 
+export async function isfilePathExist(fsPath: string): Promise<boolean> {
```

Comment:
Please leverage `fse.pathExists` instead of writing your own function. See here for more info: https://github.com/Microsoft/vscode-azurefunctions/blob/3c65d5596bc7c0ff8f2ac106793690610052a5af/src/utils/fs.ts#L38

I thought we were already using 'fs-extra' in all of our repos, but I guess we haven't started using it here yet. You'll have to add it to the package.json